<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TargetDataDlg</class>
 <widget class="QDialog" name="TargetDataDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>加载目标数据</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>加载tmp数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>加载TRI数据</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tmp">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_TRI">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>490</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>330</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>原点坐标（经纬度坐标）</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditX"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>纬度：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditY"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>高度：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditZ"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>经度：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>度</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>度</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>米</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_ToRadio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>291</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>转为辐射值</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_fen"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>起始波长：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_fa"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>终止波长：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>发射率：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_start"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>谱分辨率：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_End"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>um</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>um</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>（波数）1/cm</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_LoadJet">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>尾焰文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_LoadJet">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Plt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>加载Plt</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Plt">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_CustomJet">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>531</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>尾焰自定义位置</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="pushButton_RemoveJet">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除尾焰</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AddJet">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>添加尾焰</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_JetPara">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>509</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>TargetDataDlg</receiver>
   <slot>slotButtonClicked_tmp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>TargetDataDlg</receiver>
   <slot>slotButtonClicked_TRI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TargetDataDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TargetDataDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_LoadJet</sender>
   <signal>clicked()</signal>
   <receiver>TargetDataDlg</receiver>
   <slot>slotButtonClicked_LoadJet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Plt</sender>
   <signal>clicked()</signal>
   <receiver>TargetDataDlg</receiver>
   <slot>slotButtonClicked_LoadPlt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotButtonClicked_tmp()</slot>
  <slot>slotButtonClicked_TRI()</slot>
  <slot>slotButtonClicked_LoadJet()</slot>
  <slot>slotButtonClicked_LoadPlt()</slot>
 </slots>
</ui>
